src/packages/core/components/icon/icon.element.ts

```diff
@@ -31,14 +41,28 @@ export class UmbIconElement extends UmbLitElement {
 		this._color = alias ? (variable ? `--uui-icon-color: var(${variable})` : `--uui-icon-color: ${alias}`) : undefined;
 	}
 
+	#setFallbackColorStyle(value: string) {
+		const alias = value.replace('color-', '');
+		const variable = extractUmbColorVariable(alias);
+		this._fallbackColor = alias
+			? variable
+				? `--uui-icon-color: var(${variable})`
+				: `--uui-icon-color: ${alias}`
+			: undefined;
+	}
```

Comment:
This seems to do almost the same thing as `#setColorStyle`, consider unifying the logic in a common function.

Reply:
There we go 😊